string action
geometry_msgs/PoseStamped poop_location
---
uint8 result
